src/genie/libs/parser/iosxr/tests/ShowBfdSession/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,153 @@
+expected_output= {
+    "interface": {
+        "Gi0/0/0/15.103": {
```

Comment:
we usually convert to full interface name by using `convert_intf_name` function. please do the same.
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/utils/common.py#L485

Reply:
fixed using convert_intf_name function.